automation_tools/satellite6/upgrade/tests/utils.py

```diff
@@ -0,0 +1,152 @@
+import os
+import pickle
+import sys
+import time
+
+from automation_tools import manage_daemon
+from automation_tools.bz import bz_bug_is_open
+from automation_tools.satellite6.hammer import (
+    get_attribute_value, hammer,
+    hammer_info_via_name
+)
+from automation_tools.satellite6.upgrade.tasks import (
+    create_rhevm_instance,
+    delete_rhevm_instance,
+    get_rhevm_client,
+    remove_all_docker_containers,
+    wait_till_rhev_instance_status
+)
+from automation_tools.satellite6.upgrade.tasks import docker_execute_command
+from automation_tools.satellite6.upgrade.tools import logger
+from fabric.api import execute, run
+
+rpm1 = 'https://inecas.fedorapeople.org/fakerepos/zoo3/bear-4.1-1.noarch.rpm'
+rpm2 = 'https://inecas.fedorapeople.org/fakerepos/zoo3/camel-0.1-1.noarch.rpm'
+data = {}
+rhev_template = 'sat6-docker-upgrade-template'
+
+logger = logger()
+
+
+def create_dict(dict):
+    data.update(dict)
+    print data
+    with open('{}.pkcl'.format('datastore'), 'wb') as pref:
+        pickle.dump(data, pref)
+
+
+def get_dict():
+    print data
+
+
+def generate_satellite_docker_clients_on_rhevm_upgrade(
+        client_os, clients_count, ak):
+    """Generates satellite clients on docker as containers
+
+    :param string client_os: Client OS of which client to be generated
+        e.g: rhel6, rhel7
+    :param string clients_count: No of clients to generate
+    :param string ak : Activation key name, to register clients
+
+    Environment Variables:
+
+    RHEV_SAT_HOST
+        The satellite hostname for which clients to be generated and
+        registered
+    RHEV_CLIENT_AK
+        The AK using which client will be registered to satellite
+    """
+    if int(clients_count) == 0:
+        logger.warning(
+            'Clients count to generate on Docker should be atleast 1 !')
+        sys.exit(1)
+    satellite_hostname = get_satellite_host()  # specify satellite hostname
+    result = {}
+    for count in range(int(clients_count)):
```

Comment:
int(clients_count) is being repeated, it looks like worth creating a variable storing the value.